lib/puppet/reports/servicenow.rb

```diff
@@ -21,11 +21,13 @@ def process
 
   def process_event_management(settings_hash)
     event_data = {
-      'source'   => 'Puppet',
-      'type'     => 'node_report',
-      # 5 => 'OK' severity
-      'severity' => '5',
-      'node'     => host,
+      'source'      => 'Puppet',
+      'type'        => 'node_report',
+      # 5           => 'OK' severity
+      'severity'    => '5',
+      'node'        => host,
+      # Source Instance is sent as event_class in the api
+      'event_class' => master_used,
```

Comment:
Such a weird name for source instance. But this looks fine. Could you add an assertion for this in the acceptance tests? Something like `expect(event['event_class']).to match(Regexp.new(Regexp.escape(<master_fqdn>:<expected_port>)))`

Reply:
Added a test but this field will be nil for this test because master_used isn't set on puppet apply runs.
https://puppet.com/docs/puppet/6.17/format_report.html. I think we may want to use something else if this is a deal breaker.